bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/FileInfo.java

```diff
@@ -112,6 +114,29 @@ public long getSizeSinceLastwrite() {
         return sizeSinceLastwrite;
     }
 
+    public ByteBuffer getExplicitLac() {
```

Comment:
These are two different things. One is on disk cache, and other one is explicitLAC; My vote is keep them separate, as some folks may not even use explicitLAC. Later if needed we can merge them.